### DISCUSSION TITLE
[Projects beta] Lack of websocket / live update is painful

### DISCUSSION THREAD

**Discussion by elyobo** [community/community#6508] [Projects and Issues] [answered]
The need to do a full refresh to pull in changes made by other users on the same project view, or other users using other interfaces (e.g. on the issues themselves) is rather painful; are these plans to make this update live?

**Comment by ynouri** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/5738?

**Reply by elyobo** [reply]
Yeah, looks like it - I did search but "hot reload" is something completely different to me and that was not a search that I tried!

**Reply by azenMatt** [reply]
Just updated the other issue - but incase you don't see it there we are putting the finishing touches to this and it will be with you pretty soon

**Reply by elyobo** [reply]
:tada: Thrilled to hear it, thanks, it will make it much easier to use in team meetings!
